src/main/java/com.gigssandbox/GigsRepository.java

```diff
@@ -0,0 +1,108 @@
+package com.gigssandbox;
+
+import com.gigssandbox.entities.Band;
+import com.gigssandbox.entities.Gig;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class GigsRepository {
+    private static final String DB_URL = "jdbc:postgresql://localhost:5432/postgres";
+    private static final String USER = "postgres";
+    private static final String PASS = "postgrespass1488";
+
+    public List<Gig> getGigsList(String... group) {
+        try (Connection connection = DriverManager.getConnection(DB_URL, USER, PASS); Statement statement = connection.createStatement();PreparedStatement statementWithParam = connection.prepareStatement("SELECT * FROM gigs WHERE date >= CURRENT_DATE AND (headliner = ? OR support = ?)")) {
+            ResultSet resultSet;
+            if (group.length == 1) {
+                statementWithParam.setString(1, group[0]);
+                statementWithParam.setString(2, group[0]);
+                resultSet = statementWithParam.executeQuery();
+            }
+            else {
+                resultSet = statement.executeQuery("SELECT * FROM gigs WHERE date >= CURRENT_DATE");
+            }
+            List<Gig> gigs = new ArrayList<>();
+            while (resultSet.next()) {
+                gigs.add(Gig.builder()
+                        .id(resultSet.getInt(1))
+                        .headliner(resultSet.getString(2))
+                        .support(resultSet.getString(3))
+                        .date(resultSet.getTimestamp(4).toLocalDateTime())
+                        .location(resultSet.getString(5))
+                        .build());
+            }
+            return gigs;
+        } catch (
+                SQLException e) {
+            e.printStackTrace();
+        }
+        return Collections.emptyList();
+    }
+
+    public List<Band> getAllBandsList() {
```

Comment:
this method is not used. You must add methods as soon as they are required. Remove it